src/store/modules/app.ts

```diff
@@ -153,14 +162,31 @@ const actions = {
   getProposal: async ({ commit }, { space, id }) => {
     commit('GET_PROPOSAL_REQUEST');
     try {
-      const provider = getProvider(space.network);
       console.time('getProposal.data');
+      const response = await Promise.all([ipfsGet(gateway, id)]);
```

Comment:
We should load in same time the list of the votes from the hub (we don't need to wait for the proposal to resolve)

Reply:
Can we include the other ones like `getPower` and `getProposals` too?

Reply:
I'm not sure what is the need of `getProposals` here? I prefer not mix the functions here. Maybe you are answering to this https://github.com/snapshot-labs/snapshot/pull/228#discussion_r567449292 ? If that's the case yes you can add these functions on the helper file too.

Reply:
Oh, yes your right I was replying to the wrong comment.